CIP-DRep-Credential/README.md

```diff
@@ -0,0 +1,393 @@
+- Self-reported data (e.g., a DRep’s focus areas or a quick rationale) is possible but should remain concise to avoid high on-chain storage costs.
+
+**Off-Chain**
+
+- Large or frequently updated content (e.g., multi-paragraph mission statements, proposal PDFs, rich media) is stored off-ledger.
+- References take the form of a **URL + blake2b-256 hash** (per CIP-119 or CIP-108), allowing client apps to verify integrity while avoiding ledger bloat.
+
+### 4. Governance NFT Types
+
+All governance NFTs proposed here share a **new CIP-67 label** (placeholder `(1694)`). Example asset names:
+
+- `"(1694)drepCredential-<dRepAlias>"`
+- `"(1694)ballotNote-<proposalId>"`
+- `"(1694)endorsement-<endorserKey>-<dRepId>"`
+
+#### 4.1 DRep Credential
+
+**Minted by:** The DRep or an authorized delegate.  
+**Purpose:**
+
+- Captures key DRep info on-chain (e.g., `dRepId`, name, roles).
+- Optionally anchors extended data via CIP-119.
+- Reflects the “DRep Credential Form” mockup fields.
+
+**Core Field:**
+
+- `dRepId` (CIP-1694 hash)
+
+**Recommended Optional Fields (Self-Reported):**
+
+- **Title/Role**: Short descriptor (e.g., “DeFi Developer”).
+- **Name (or Alias)**, **Profile Image URL**, **Website**, **Mission/Goals**.
+- **Primary Focus Areas** (e.g., DeFi, Governance, Education).
+- **Roles** (SPO, Dev/Builder, Community Manager, etc.).
+- **Sectors** (NFTs, Identity, Gaming, DAO Tools).
+- **Social Links** (Twitter handle, Discord handle).
+- **Off-Chain Anchor** (`cip119AnchorUrl`, `cip119AnchorHash`) for extended profiles.
+
+**Minimal JSON Example:**
+
+```jsonc
+{
+  "version": 1,
+  "dRepId": "b2f0...20fb",
+  "titleOrRole": "DeFi Developer",
+  "nameOrAlias": "Alice The Builder",
+  "missionOrGoals": "Promoting transparent DeFi governance on Cardano",
+  "primaryFocusAreas": ["DeFi", "Governance"],
+  "roles": ["Dev/Builder", "Community Manager"],
+  "website": "https://mydrepwebsite.com",
+  "twitterHandle": "@aliceBuilder",
+  "cip119AnchorUrl": "ipfs://bafy123...",
+  "cip119AnchorHash": "blake2b256:abc123..."
+}
+```
+
+#### 4.2 Ballot Note
+
+**Minted by:** A DRep publishing a stance on a proposal.  
+**Purpose:**
+
+- Declares the DRep’s intended vote (`Yes`, `No`, or `Abstain`) for a CIP-1694 proposal.
+- Can include a short rationale or reference CIP-108 anchor for deeper explanation.
+
+**Core Fields:**
+
+- `type = "ballotNote"`
+- `dRepId` (CIP-1694)
+- `proposalId` (CIP-1694)
+- `voteChoice`: `"Yes" | "No" | "Abstain"`
+
+**Optional Fields:**
+
+- `rationale`, `timestamp`
+- CIP-108 anchor (`anchorUrl`, `anchorHash`)
+
+**Minimal JSON Example:**
+
+```jsonc
+{
+  "version": 1,
+  "type": "ballotNote",
+  "dRepId": "b2f0...20fb",
+  "proposalId": "abcd1234+0",
+  "voteChoice": "Yes",
+  "rationale": "Supports Cardano's long-term growth",
+  "timestamp": "2025-03-10T12:00:00Z",
+  "anchorUrl": "ipfs://proposalDoc",
+  "anchorHash": "blake2b256:def456..."
+}
+```
+
+#### 4.3 Endorsement
```

Comment:
Im not sure what the motivation for stake-based endorsement is?
would it not be easier to follow chain and look at the vote delegations from stake accounts to DRep credentials?

Reply:
I do not know how to answer this one!

Reply:
to rephrase the question

all delegations to DReps are public via the chain, is there a need to track this via NFTs also?